module/signature/aggregation.go

```diff
@@ -89,13 +75,10 @@ func (ap *AggregationProvider) Sign(msg []byte) (crypto.Signature, error) {
 // Aggregate will aggregate the given signatures into one aggregated signature.
 func (ap *AggregationProvider) Aggregate(sigs []crypto.Signature) (crypto.Signature, error) {
 
-	// NOTE: the current implementation simply concatenates all signatures; this
-	// will be replace by real AggregationProvider signature aggregation once available
-	c := &Combiner{}
-	sig, err := c.Join(sigs...)
+	// BLS aggregation
+	sig, err := crypto.AggregateBLSSignatures(sigs)
 	if err != nil {
-		return nil, fmt.Errorf("could not combine signatures: %w", err)
+		return nil, fmt.Errorf("could not aggregate signatures: %w", err)
```

Comment:
This implementation specifically uses BSL signatures. Lets reflect this in the error message. 
```suggestion
		return nil, fmt.Errorf("could not aggregate BLS signatures: %w", err)
```